terraform/main.tf

```diff
@@ -305,27 +308,27 @@ data "aws_iam_role" "execution-role" {
 }
 
 resource "aws_ecs_task_definition" "c11-railway-tracker-dashboard-ECS-task-def-tf" {
-  family = "c11-railway-tracker-dashboard-ECS-task-def-tf"
+  family                   = "c11-railway-tracker-dashboard-ECS-task-def-tf"
   requires_compatibilities = ["FARGATE"]
-  network_mode = "awsvpc"
-  execution_role_arn = data.aws_iam_role.execution-role.arn
-  cpu = 1024
-  memory = 2048
-  container_definitions = jsonencode([
+  network_mode             = "awsvpc"
+  execution_role_arn       = data.aws_iam_role.execution-role.arn
+  cpu                      = 1024
+  memory                   = 2048
+  container_definitions    = jsonencode([
     {
-      name = "c11-railway-tracker-dashboard-ECS-task-def-tf"
-      image = "129033205317.dkr.ecr.eu-west-2.amazonaws.com/c11-railway-tracker-dashboard-erc:latest"
-      cpu = 10
-      memory = 512
-      essential = true
+      name         = "c11-railway-tracker-dashboard-ECS-task-def-tf"
+      image        = "129033205317.dkr.ecr.eu-west-2.amazonaws.com/c11-railway-tracker-dashboard-erc:latest"
+      cpu          = 10
+      memory       = 512
+      essential    = true
       portMappings = [
         {
             containerPort = 80
-            hostPort = 80
+            hostPort      = 80
         },
         {
             containerPort = 8501
-            hostPort = 8501       
+            hostPort      = 8501       
```

Comment:
Can you not specify the protocol used here?

Reply:
"Port mappings
For task definitions that use the awsvpc network mode, only specify the containerPort. The hostPort can be kept blank or the same value as the containerPort."
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_portmappings